backend/api/v1/endpoints/research.py

```
@@ -0,0 +1,373 @@
+        db.add(new_research)
+        db.commit()
+        db.refresh(new_research)
+
+        return {"message": "Research work posted successfully", "research_id": new_research.id}
+
+    except Exception as e:
+        logging.error(f"Error posting research: {str(e)}")
+        raise HTTPException(status_code=500, detail= give_error)
+
+@router.post("/request-collaboration/{research_id}/")
+def request_collaboration(
```

Ruff (0.8.2)

Missing return type annotation for public function `request_collaboration`

(ANN201)